src/pyfmi/master.pyx

```diff
@@ -299,8 +305,41 @@ class MasterAlgOptions(OptionBase):
             'der'. Can also be a list. Note that there should be one
             filter for each model.
             Default: None
+            
+        logging --
+            If True, store additional debug data during the simulation.
+            The option is currently only useful for internal purposes.
+            Default: False
+        
+        store_step_before_update --
+            If True, store additionally the values in the underlying 
+            FMUs to the result file before data has been exchange 
+            between the connected models. The values will always be 
+            stored after data has been exhanged between the connected
+            models.
+            Default: False
+        
+        block_initialization_type --
+            Specifies which algorithm should be used to find the best
+            grouping of input/outputs. This only has an effect when the
+            option 'block_initialization' is set to True. The available
+            values are: "greedy", "simply", "grouping". Each has their
+            pros and cons, for best performance, please test.
+            Default: "greedy"
 
-
+        block_initialization_order --
+            If set, manually overrrides the order in which the connected
+            models should be initialized. This only has an effect when
+            the option 'block_initialization' is set to True.
+            Default: None
```

Comment:
Also here it is unclear which format this should be specified in.

Reply:
Looking at this again, I believe this should be an experimental option. I've removed the description and marked the option as experimental instead.